cinema/models.py

```diff
@@ -1,9 +1,46 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=127)
+    last_name = models.CharField(max_length=127)
+
+    def __str__(self) -> str:
+        return f"{self.first_name} {self.last_name}"
+
+
+class Genre(models.Model):
+    name = models.CharField(
+        max_length=255,
+        unique=True
+    )
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.SmallIntegerField()
+    seats_in_row = models.SmallIntegerField()
+
+    def __str__(self) -> str:
+        return self.name
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
+    actors = models.ManyToManyField(
+        Actor,
+        related_name="movies",
+        blank=True,
+    )
+    genres = models.ManyToManyField(
+        Genre,
+        related_name="movies",
+        blank=True,
+    )
```

Comment:
Sometimes when movie is small and or either you don't know actors and genre is 2 much underground it could happen, so I disagree)
And dummy movie is also a good point sometimes.